device_vivalto3gvn.mk

```diff
@@ -16,6 +16,28 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/full_base_telephony.mk)
 # Inherit from fortuna3g device
 $(call inherit-product, $(LOCAL_PATH)/device.mk)
 
+# Keylayouts
+PRODUCT_COPY_FILES += \
+	device/samsung/vivalto3gvn/keylayouts/ist30xx_ts_input.kl:system/usr/keylayout/ist30xx_ts_input.kl
```

Comment:
I do not have time to commit so someone please add the following contents
to that file

key 158 BACK VIRTUAL
key 254 APP_SWITCH VIRTUAL
On Dec 29, 2015 8:32 PM, "K_TECH" <notifications@github.com> wrote:

> In device_vivalto3gvn.mk
> <https://github.com/vivalto/android_device_samsung_vivalto3gvn/pull/2#discussion_r48540246>
> :
>
> > @@ -16,6 +16,28 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/full_base_telephony.mk)
> >  # Inherit from fortuna3g device
> >  $(call inherit-product, $(LOCAL_PATH)/device.mk)
> >
> > +# Keylayouts
> > +PRODUCT_COPY_FILES += \
> > +	device/samsung/vivalto3gvn/keylayouts/ist30xx_ts_input.kl:system/usr/keylayout/ist30xx_ts_input.kl
>
> -1
> there are no ist30xx_ts_input.kl in this tree
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vivalto/android_device_samsung_vivalto3gvn/pull/2/files#r48540246>
> .
>